requirements.txt

```diff
@@ -0,0 +1,9 @@
+elasticsearch==6.3.1
```

Comment:
I think we are using opensearch, why do we need to declare ES?

Reply:
Currently we still using ES client, I've created an issue to replace es client with opensearch client. https://github.com/oss-compass/compass-metrics-model/issues/75

Reply:
> I think we are using opensearch, why do we need to declare ES?



> I think we are using opensearch, why do we need to declare ES?

The opensearch client is currently in use, but the es client has not been removed.